OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -58,6 +58,9 @@ public sealed class HackyAIInfo : IBotInfo, ITraitInfo
 		[Desc("How long to wait (in ticks) between structure production checks ticks when actively building things.")]
 		public readonly int StructureProductionActiveDelay = 10;
 
+		[Desc("Limits the AI to issuing this many orders per tick. Orders that cannot be issued in this tick are queued for the next tick.")]
+		public readonly int MaxOrdersPerTick = 1;
```

Comment:
The order queue grows unboundedly as soon as the average order count per tick gets larger than this.  I found that the AI would grind to a halt in the late game until I turned this up.

That obviously isn't a great solution (since it just defers the problem until they have sufficiently more units, and also hurts perf in the early game) - can we dynamically scale this based on the current queue size instead?